### DISCUSSION TITLE
请问Table标签有没有属性可以来作为标记位置

### DISCUSSION THREAD

**Discussion by Sole-BadGuy** [wangeditor-next/wangEditor-next#398] [General]
## 问题描述

请问Table标签有没有属性可以来作为标记位置 比如后端返回的html模板里包含3个table我需要通过标记位来判断哪个table被删除了 我使用id加到table上会被刷掉

## wangEditor 版本
最新

## 是否查阅了文档 ？
是

**Comment by cycleccc** [comment]
id 是 wangEditor 内部 vnode diff 更新用的。
自定义插件，在自定义插件的标签里定义一个自定义属性可以实现。

**Comment by cycleccc** [comment]
table 的 id 为内部 vnode diff 使用；业务标记应由自定义元素的自定义属性承载。本讨论现关闭。
